config/api.yml

```diff
@@ -422,8 +422,16 @@
     :options:
     - :collection
     - :subcollection
-    :methods: *70174834086080
+    :methods: *70174834085620
     :klass: ChargebackRateDetail
+    :collection_actions:
+      :post:
+      - :name: create
+    :resource_actions:
+      :post:
+      - :name: edit
+      :delete:
+      - :name: delete
   :reports:
```

Comment:
The new actions defined here are missing the identifier entries.  We need to make sure the proper Product feature names are defined here (look @ the UI for role parity) here for RBAC otherwise anyone can run these actions.